server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -85,6 +85,12 @@ grant codeBase "${codebase.zstd-jni}" {
   permission java.lang.RuntimePermission "loadLibrary.*";
 };
 
+grant codeBase "${codebase.bcprov-jdk15to18}" {
```

Comment:
Do we need this for jdk 11?

Reply:
This is the way that bouncycastle releases artifacts. Until last month the securityplugin was using [jdk15on](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on), but then switched to [jdk15to18](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15to18) (PR where it was changed: https://github.com/opensearch-project/security/pull/2901). I plan to open an issue about using [jdk18on](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on).

Reply:
No, this is due to https://github.com/opensearch-project/OpenSearch/pull/9289, the BC has moved to core and now we need this permission to manage security providers.